bom/runtime/pom.xml

```diff
@@ -117,6 +117,7 @@
         <assertj.version>3.12.2</assertj.version>
         <json-smart.version>2.3</json-smart.version>
         <infinispan.version>10.0.0.Beta5</infinispan.version>
+        <infinispan.protostream.version>4.3.0.Alpha9</infinispan.protostream.version>
```

Comment:
Alpha10 was released yesterday.

Reply:
I will update.

Reply:
Actually I can't update this until we release a new build of ISPN. It has to match the version that ISPN defines. I have reverted it back to Alpha9 until then.